code/modules/jobs/job_types/enclave.dm

```diff
@@ -416,7 +416,7 @@
 		/obj/item/grenade/chem_grenade/cleaner = 1,
 		/obj/item/pda = 1,
 		/obj/item/gun/energy/gammagun = 1,
-		/obj/item/stock_parts/cell/ammo/ec = 2,
+		/obj/item/stock_parts/cell/ammo/mfc = 2,
```

Comment:
why

Reply:
messed up before, got it mixed around. gamma guns are run off of MFCs and not ECs. this fixes the ammo the scientists spawn with